agency/README.md

```diff
@@ -164,7 +173,7 @@ _No content returned if no vehicle matching `device_id` is found._
 
 ## Vehicle - Event
 
-The vehicle `/event` endpoint allows the Provider to control the state of the vehicle including deregister a vehicle from the fleet.
+The vehicle `/event` endpoint allows the Provider to control the state of the vehicle.
```

Comment:
Should 'including deregister a vehicle from the fleet' remain?

Reply:
`deregister` isn't really a word anymore in MDS, but we could include `decomission` or something here.